tests/integrations/gcp/test_gcp.py

```diff
@@ -102,16 +112,24 @@ def inner(code, subprocess_kwargs=()):
             )
 
             stream = os.popen("python {}/main.py".format(tmpdir))
-            event = stream.read()
-            event = json.loads(event[len("EVENT: ") :])
-
-        return event
+            stream_data = stream.read()
+            delimeter = "EVENT: "
+            if "ENVELOPE:" in stream_data:
+                delimeter = delimeter + "|ENVELOPE: "
+            stream_list = re.split(delimeter, stream_data)
```

Comment:
This seems odd, can't we do the same as in AWS?

Reply:
I checked and I did it as per AWS, changes will be in next commit.